ObjectMapper/Core/Mapper.swift

```diff
@@ -102,6 +102,13 @@ public final class Mapper<N: BaseMappable> {
 			}
 		}
 
```

Comment:
For consistency I should add a comment like `// check if object is ImmutableMappable`, although I would argue that the code is self-commenting.